ext/http/http_next.rs

```diff
@@ -208,8 +209,8 @@ pub async fn op_http_upgrade_websocket_next(
   ws_create_server_stream(&mut state.borrow_mut(), stream, bytes)
 }
 
-#[op(fast)]
-pub fn op_http_set_promise_complete(slab_id: SlabId, status: u16) {
+#[op2(fast)]
+pub fn op_http_set_promise_complete(#[smi] slab_id: SlabId, status: u16) {
```

Comment:
This appears to be buggy. Can `u16` be actually used in fast calls?

Reply:
It should not be a problem. It is actually the only way to use integer aliases. Let me take a closer look

Reply:
I think it's a non-issue. I see that it's later cast to u32 by the macro itself.